commentTone/pom.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.whz</groupId>
+    <artifactId>commentTone</artifactId>
+    <version>1.0.0</version>
+
+    <properties>
+        <jdk.version>11</jdk.version>
+        <maven.compiler.target>${jdk.version}</maven.compiler.target>
+        <maven.compiler.source>${jdk.version}</maven.compiler.source>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <spotless.version>2.5.0</spotless.version>
+        <maven-assembly-plugin.version>3.3.0</maven-assembly-plugin.version>
+        <rabbitmq.version>5.10.0</rabbitmq.version>
+        <gson.version>2.8.2</gson.version>
+
+        <junit.version>5.6.0</junit.version>
+        <mockito.version>3.5.15</mockito.version>
+        <restassured.version>4.3.1</restassured.version>
```

Comment:
This dependencies are for tests in future

Reply:
You don't need the dependency `restassured` because you don't have a REST interface. Otherwise, I would add some tests to this PR, as we don't have much time left in the future.

Reply:
Dependencies for test are removed. Then I'll do tests like Simon's